### PR TITLE
Topic/update ucx desc in spec and readme v1.5.x

### PR DESCRIPTION
**Pull request by yosefe**
## What
_Describe what this PR is doing._ 

## Why ?
_Justification for the PR. If there is existing issue/bug please reference. For
bug fixes why and what can be merged in a single item._

## How ?
_It is optional but for complex PRs please provide information about the design,
architecture, approach, etc._

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,5 +1,11 @@
-UCX is a communication library implementing a high-performance messaging
-layer for MPI, PGAS, and RPC frameworks.
+UCX stands for Unified Communication X. UCX provides an optimized communication
+layer for Message Passing (MPI), PGAS/OpenSHMEM libraries and RPC/data-centric
+applications. UCX utilizes high-speed networks, such as RDMA (InfiniBand, RoCE,
+etc), Cray Gemini or Aries, for inter-node communication. If no such network is
+available, TCP is used instead. UCX supports efficient transfer of data in
+either main memory (RAM) or GPU memory (through CUDA and RoCM libraries).
```
Comment (bbenton) [inline README]: s/RoCM/ROCm/

File: ucx.spec.in [hunk 2]
```diff
@@ -19,12 +19,14 @@ BuildRequires: numactl-devel libibverbs-devel
...
-Future versions will support also TCP for inter-node, to lift that hardware
-dependency.
-In addition, the library can be used for intra-node communication by leveraging
-the following shared memory mechanisms: posix. sysv, cma, knem, xpmem.
+UCX stands for Unified Communication X. UCX provides an optimized communication
+layer for Message Passing (MPI), PGAS/OpenSHMEM libraries and RPC/data-centric
+applications. UCX utilizes high-speed networks, such as RDMA (InfiniBand, RoCE,
+etc), Cray Gemini or Aries, for inter-node communication. If no such network is
+available, TCP is used instead. UCX supports efficient transfer of data in
+either main memory (RAM) or GPU memory (through CUDA and RoCM libraries).
```
Comment (bbenton) [inline ucx.spec.in]: s/RoCM/ROCm/

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8693/ for details (Mellanox internal link).

File: README (hunk 1 above)
Comment (shamisp) [reply README]: @yosefe Ok. I guess has to be ported to master as well

**Comment by yosefe** [comment]
@bbenton fixed

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8697/ for details (Mellanox internal link).

**Comment by bbenton** [comment]
:+1:  Thanks @yosefe

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8709/ for details (Mellanox internal link).

**Pull request merged**